app/models/state_file_md_intake.rb

```diff
@@ -96,6 +104,7 @@
 #  index_state_file_md_intakes_on_spouse_state_id_id   (spouse_state_id_id)
 #
 class StateFileMdIntake < StateFileBaseIntake
+  self.ignored_columns = [:df_data_import_failed, :account_holder_name, :bank_name]
```

Comment:
same thing from other comments about `df_data_import_failed` and having `bank_name` on base intake